andela/travel_tool_front
# #161979747 Restrict React/React-dom versions to ~16.5.1

#### What does this PR do?
Restricts React/React-dom versions to `~16.5.1`

#### Description of Task to be completed?

- replace react/react-dom versions with `~16.5.1`
- update the `yarn.lock` file

#### How should this be manually tested?
- Clone the repo
    `git clone https://github.com/andela/travel_tool_front.git`
 - Switch to the project directory
   `cd travel_tool_front`
 - Checkout the **ch-restrict-react-version-161979747** branch
   `git checkout ch-restrict-react-version-161979747`
 - Install all dependencies
   `yarn install`
 - Start the [backend](https://github.com/andela/travel_tool_back) server
 - Start the server:
    - `make start` *only if you have docker*
    - `yarn start` *if you do not have docker*
 - Navigate to your browser and click the **New Request** button.
  It is expected that you can type in the new request form.

#### Any background context you want to provide?
You will need yarn version 1.12.x to run `yarn install`.

#### What are the relevant pivotal tracker stories?
- [#161979747](https://www.pivotaltracker.com/story/show/161979747)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
